src/pages/onboarding/create.vue

```diff
@@ -379,6 +392,20 @@ export default {
 
     async removeTeamMember (username) {
       this.form.members = this.form.members.filter((obj) => obj.username !== username)
+    },
+
+    goToStep ({ index }) {
+      this.activeStepIndex = index
+      if (this.$q.screen.gt.md) {
+        switch (index) {
```

Comment:
You can add step names to the steps array

https://github.com/hypha-dao/dho-web-client/blob/5ff7b4f5fff2d9b4cb2d4c523d12b352ac86d675/src/pages/onboarding/create.vue#L148

so that you can do 
`this.scrollToNextStep(steps[activeStepIndex].id)`